packages/NodeTypeResolver/Node/AttributeKey.php

```diff
@@ -71,6 +71,8 @@ final class AttributeKey
      * @internal of php-parser, do not change
      * @see https://github.com/nikic/PHP-Parser/pull/681/files
      * @var string
+     *
+     * @api for BC layer now
```

Comment:
comment here is needed that `parent` can still be enabled by use:

```neon
conditionalTags:
    PhpParser\NodeVisitor\ParentConnectingVisitor:
        phpstan.parser.richParserNodeVisitor: true
```

in custom phpstan config for : 

```php
$rectorConfig->phpstanConfig(__DIR__ . '/phpstan.neon.dist');
```

with cautions that will not be reliable on attribute changed after, see removed config at PR:

- https://github.com/rectorphp/rector-src/pull/4415#pullrequestreview-1512291349 

for init parent set.

Reply:
Good point, I've added the link to this commen to have easy-to-read context :+1: